app/modules/notifications/models.py

```diff
@@ -217,16 +218,19 @@ def set_individual_merge(self, individuals, encounters, request_data):
         for indiv in individuals:
             ind_data = {'guid': indiv.guid, 'primaryName': indiv.get_primary_name()}
             self.data['individual_list'].append(ind_data)
+            source_names.append(indiv.get_primary_name())
             if 'target_individual_guid' not in self.data:
                 self.data['target_individual_guid'] = str(indiv.guid)
                 self.data['target_individual_name'] = indiv.get_primary_name()
-            else:
-                source_names.append(indiv.get_primary_name())
+
         self.data['source_names_list'] = ', '.join(source_names)
         self.data['encounter_list'] = []
         for enc in encounters:
             self.data['encounter_list'].append(enc.guid)
         self.data['request_id'] = request_data.get('id')
+        deadline_datetime = request_data.get('deadline')
+        if deadline_datetime:
+            self.data['deadline'] = deadline_datetime.isoformat()
```

Comment:
I'm seeing a lot of foo.isoformat() + '+00:00' (particularly on tests related to individuals) and foo.isoformat() + 'Z' in the code base and vaguely remember this being important. Might be worth checking with someone who acutally knows whether this change is needed in this case.

Reply:
This is a valid question and I think it's correct that something should be added but there is a mix in the code where sometimes it is ad sometimes it isn't. @naknomum can usually be relied upon to have an opinion about this :)

Reply:
After convo w Jon, added 'Z' to the string. Thanks fellas!